src/commands/set_wo.json

```diff
@@ -0,0 +1,97 @@
+{
+    "SET_WO": {
```

Comment:
when you update from unstable, don't forget to bring in the new tips from SET

Reply:
i don't see SET have the tips? https://github.com/redis/redis/blob/unstable/src/commands/set.json

Reply:
sorry.. my bad